components/icon/icon.css

```diff
@@ -0,0 +1,17 @@
+@tailwind components;
+
+@layer components {
+  .c-icon {
+    display: inline-flex;
+    fill: currentColor;
```

Comment:
this plain css will be replaced with tailwind classes. This will be done in this task: https://github.com/ost-cas-fee-adv-23-24/design-system-component-library-football-fanatics/issues/4